src/pages/functional-testing-framework/metadata.md

```diff
@@ -264,6 +270,157 @@ _Catalog/Data/CategoryData.xml_:
+</entity>
+```
+
+- `type="company_relation"` points to the operation with `dataType=company_relation"` and `type="create"` in the _Metadata_ directory.
+- `<requiredEntity type="company_id">CompanyId</requiredEntity>` points to the data entity with `name="CompanyId"` and `type="company_id"`.
+
+_CompanyRelations/Data/CompanyIdData.xml_:
+
+```xml
+<entity name="CompanyId" type="company_id">
+    <var key="company_id" entityType="company2" entityKey="id"/>
+</entity>
+```
+
+- `type="company_id"` points to the operation with `dataType=company_id"` and `type="create"` in the _Metadata_ directory.
+- The `company_id` will only get the `id` from the company with `type="company2"`
+
+_CompanyRelation/Metadata/CompanyIdMeta.xml_:
+
+```xml
+<operation name="CreateCompanyId" dataType="company_id" type="create">
+    <field key="company_id">integer</field>
+</operation>
+```
+
+`<field key="company_id">integer</field>` will create an object with a key `company_id` with a value of type integer.
+
+```json
+{
+  "company_id": 1
+}
+```
+
+_CompanyRelation/Metadata/CompanyRelationMeta.xml_:
+
+```xml
+<operation name="CreateCompanyRelation" dataType="company_relation" type="create" auth="adminOauth" url="/V1/company/{company.id}/relations" method="POST">
+    <contentType>application/json</contentType>
+    <array key="relations">
+        <value>company_id</value>
+    </array>
+</operation>
+```
+
+As a result, the Functional Testing Framework sends a POST request with an array of company_ids in the body to the `https://example.com/rest/V1/company/{company.id}/relations`. Currently, the Functional Testing Framework supports sending only one company as a child company for a parent.
+
+- `{company.id}` in the url comes from the test `<requiredEntity createDataKey="parentCompany"/>` in OneCompanyRelation that has `type="company"`
+- `<array key="relations">` will create an array with the key relations
+- `<value>company_id</value>` will get the data from the operation with `dataType="company_id"`
+
+```json
+{
+  "relations": [
+    {
+      "company_id":  1
+    }
+  ]
+}
+```
+
+##### Delete a company relation
+
+The corresponding test step is:
+
+```xml
+<deleteData createDataKey="createCompanyRelation" stepKey="deleteCompanyRelation"/>
+```
+
+`createDataKey="createCompanyRelation"` will be the same as the createData with `stepKey="createCompanyRelation"` in the test. This will call the delete operation that has the same `dataType="company_relation"` as the createData.
+
+_CompanyRelation/Metadata/CompanyRelationMeta.xml_:
+
+```xml
+<operation name="DeleteCompanyRelation" dataType="company_relation" type="delete" auth="adminOauth" url="/V1/company/{company.id}/relations/{company2.id}" method="DELETE">
+    <contentType>application/json</contentType>
+</operation>
+```
+
+As a result, the Functional Testing Framework sends a DELETE request to the `https://example.com/rest/V1/company/{company.id}/relations/{company2.id}`.
```

Comment:
```suggestion
As a result, the Functional Testing Framework sends a DELETE request to `https://example.com/rest/V1/company/{company.id}/relations/{company2.id}`.
```

Reply:
any prefix of "the" where URL literal acts as the noun is slightly jarring.  really really really minor